### PR TITLE
Fixed spelling mistakes and grammar

### PR DESCRIPTION
**Pull request by abhisheksoni27**

### CODE REVIEW

**Comment by jorgebucaran** [comment]
LGTM

**Comment by Swizz** [comment]
Why all this tabs changes in the source code ?

**Comment by abhisheksoni27** [comment]
I saw them and rechecked the actual presentation of the README.md in my forked repo and it looked exactly the same. I don't know why the tabs changed, though. I use VS code. Could that have done this? :thinking: 

Should I try editing on Sublime/Atom?

**Comment by jorgebucaran** [comment]
I think we using tabs it's my fault. 😅

**Comment by abhisheksoni27** [comment]
So which convention would you like the code to follow?

**Comment by abhisheksoni27** [comment]
Also, @jbucaran, I filed an issue over at `Travis-CI` because the `node-7.7.3` test failed. (It was due to an internal network trouble in downloading the tarball. ) 

They want you to restart the build again to see if the issue has resolved. 

Could you? Here's a link to the issue I filed: [Why is Travis CI taking too long to download Node.js](https://github.com/travis-ci/travis-ci/issues/7474#issuecomment-287087496)

File: README.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ const fly = new Fly({
 
 ### Task
 
-A Task receives itself as its first argument. We choose to name the parameter `fly` simply because of convention; of course, you may call it whatever you'd like.
+A Task receives itself as its first argument. We choose to name the parameter `fly` simply because of the convention; of course, you may call it whatever you'd like.
```
Comment (lukeed) [inline README.md]: The use of `the` is wrong here, but you're right -- sounds weird here. 

Let's change to "simply as a convention;".

**Review by lukeed — CHANGES_REQUESTED** [review]

**Comment by lukeed** [comment]
Thanks @abhisheksoni27! 

Also, there's no need for a Travis issue. It halted because it was processing a lot of PRs/commits  at once. You'll see that it's now passing. :)

File: README.md (hunk 1 above)
Comment (abhisheksoni27) [reply README.md]: Well, yeah. The `the` **seems** to be referring to some ubiquitous and insurmountable convention followed by everyone.

"simply as a convention" sounds about right.

**Comment by abhisheksoni27** [comment]
@lukeed &ndash; Thanks for reviewing!

Also, you are right. They are now passing. But check this: [Travis Incident Page](https://www.traviscistatus.com/incidents/3s7jqhhzt8k1)

They think it's an upstream issue. You should tell them if you are sure it was just Travis getting overwhelmed.

**Review by lukeed — APPROVED** [review]

**Comment by lukeed** [comment]
Thanks @abhisheksoni27. I think it's something to do with my own configuration inside Travis. I'll take a look!

Thank you for proofreading! 🙏 One can only stare at the same wall of text before insanity strikes.

**Comment by abhisheksoni27** [comment]
You're welcome. :tada:

Thanks for the project, @lukeed. One can only type the same commands before automation strikes.

**Pull request merged**